support-frontend/assets/helpers/campaigns.jsx

```diff
@@ -113,14 +68,18 @@ export const campaigns: Campaigns = {
       </div>
     ),
     goalReachedCopy: null,
-    tickerJsonUrl: '/ticker.json',
     tickerType: 'unlimited',
-    cssModifiers: ['campaign-landing'],
-    contributionTypes: generateContributionTypes([
-      { contributionType: 'ONE_OFF', isDefault: true },
-    ]),
-    backgroundImage: 'https://media.guim.co.uk/de76ba8d8823325d02ff93376cfe0c39962b215d/0_0_2000_577/2000.jpg',
-    localCurrencySymbol: '$',
+    cssModifiers: ['environment-moment'],
+    localCurrencySymbol: '£',
```

Comment:
not your fault, but `localCurrencySymbol` should really be an optional field as it's only relevant to the ticker, and this is a global campaign.
And actually, there are 3 fields that are ticker-specific: `tickerJsonUrl`, `tickerType`,`localCurrencySymbol`
Is this a good time to change the campaign model to have an optional field? e.g.:
```
tickerSettings?: {
  tickerJsonUrl: string,
  tickerType: TickerType,
  localCurrencySymbol: string
}
```

Reply:
also goalReachedCopy